crates/compiler/parse/src/remove_spaces.rs

```diff
@@ -598,12 +598,103 @@ impl<'a> RemoveSpaces<'a> for StrLiteral<'a> {
     fn remove_spaces(&self, arena: &'a Bump) -> Self {
```

Comment:
Do you think it would be helpful to change the name or maybe add a doc comment? We are doing more than just removing spaces now.

Reply:
That's reasonable. I'm thinking, `Normalize::normalize` (i.e. trait and method name)